testing/test_dashboard/test_DashboardDB.py

```diff
@@ -5,6 +5,8 @@
 import pytest
 
 import qcore.constants as const
+
+sys.path.insert(0, os.path.join(os.path.abspath(os.path.dirname(__file__)), '../..'))
```

Comment:
Instead of path manipulations in python. Require that the metadata pythonfile is in the pythonpath before execution..